Meetings/2022-08-16.md

```diff
@@ -0,0 +1,66 @@
+# PlanetaryPy TC Meeting August 16th @ 12:30 PM PST / 1:30 PM MST / 2:30 PM CST / 3:30 PM EST / 20:30 UTC
+
+* Zoom Link: https://cuboulder.zoom.us/j/2789149686
+
+## Participants
+
+* Michael Aye (MA)
+* Ross Beyer (RB)
+* Andrew Annex (AA)
+
+## pending AIs
+* setup poll for new meeting time
+
+## Agenda
+
+### craterstats
+
+* Michael met with Greg about craterstats
+* Greg doesn't mind a helping hand for the installation bugs
+* To avoid complexity with import_lib we should build it for Python 3.8, when everything was in the standardlib
+* I explained to Greg how the requirements.txt file needs to balance ease of life for developer against users
+  * Lots of patch-level version pinnings make it hard to install for the user, unless it only would ever be used in its own virtual environment
+* I also recommendend against pointing the user to any specific environment, as a novice user could interpret it as that kind of environment to be mandatory, which it rarely is.
+* After we made the package installable, MA will initiate/move forward the review process.
+* Lots of ideas on how to imprive the project, by making better use of the standard library, but we leave that for later.
+ 
+### Python version
+We agree to base all things (wherever easily possible and/or under our control) on Python 3.8 for convenience, as a lot of things run 3.9 or 3.10 already, so 3.8 is a convenient but already conservative-ish standard.
```

Comment:
Oh, I must have zoned out.  I thought we were just talking about craterstats here.  Our current stance on Python version compatibility is Python 3.6 (https://planetarypy.org/packages/review-guidelines) which has now reached EOL.  I'm not sure that the "general" recommendation should be to skip to 3.8, but 3.7 only has one more year of life, so not sure.  I think we should talk about this again, just to very clearly have the conversation.